dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -39,15 +39,15 @@ ENV NJOBS="2" \
     OPAMYES="true"
 
 # Base opam is the set of base packages required by Coq
-ENV COMPILER="4.05.0"
+ENV COMPILER="4.09.0"
 
 # Common OPAM packages
 ENV BASE_OPAM="zarith.1.12 ocamlfind.1.9.1 ounit2.2.2.3 odoc.1.5.3" \
     CI_OPAM="ocamlgraph.1.8.8 yojson.1.7.0 cppo.1.6.8" \
     BASE_ONLY_OPAM="dune.2.7.1 elpi.1.14.1 stdlib-shims.0.1.0"
 
 # BASE switch; CI_OPAM contains Coq's CI dependencies.
-ENV COQIDE_OPAM="cairo2.0.6.1 lablgtk3-sourceview3.3.1.0"
+ENV COQIDE_OPAM="cairo2.0.6.1 lablgtk3-sourceview3.3.1.2"
```

Comment:
Since this got bumped should we update install.md?

Reply:
Yes, please!

Reply:
And the changelog entry should be edited to include this information.